README.md

```diff
@@ -1,16 +1,36 @@
+| [Complex Bottom Sheet](https://pxblue.github.io/patterns/overlay)              | [Link](https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/overlays/complex-bottom-sheet/index.tsx) |
+| [Data List](https://pxblue.github.io/patterns/lists)                           | [Link](https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/lists/data-list/index.tsx)               |
+| [Dynamic Stepper](https://pxblue.github.io/patterns/steppers)                  | [Link](https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/dynamic-stepper/index.tsx)               |
+| [Form Validation](https://pxblue.github.io/patterns/forms)                     | [Link](https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/form-validation/index.tsx)               |
+| [Internationalization](https://pxblue.github.io/patterns/internationalization) | [Link](https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/i18n/index.tsx)                          |
+| [Loading States](https://pxblue.github.io/patterns/empty-states)               | [Link](https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/loading-states/index.tsx)                |
+| [Multi-select List](https://pxblue.github.io/patterns/lists)                   | [Link](https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/lists/multiselect-list/index.tsx)        |
+| [Responsive Table](https://pxblue.github.io/patterns/lists)                    | [Link](https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/lists/responsive-table/index.tsx)        |
+| [Search Bar](https://pxblue.github.io/patterns/appbar)                         | [Link](https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/app-bar/search-bar/index.tsx)            |
+| [Sortable List](https://pxblue.github.io/patterns/lists)                       | [Link](https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/lists/sortable-list/index.tsx)           |
+| [Status List](https://pxblue.github.io/patterns/lists)                         | [Link](https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/lists/status-list/index.tsx)             |
 
 ## Update
+
 To update the code base, first clone the repository and make a feature branch:
+
 ```
 git clone https://github.com/pxblue/react-design-patterns.git
 cd react-design-patterns
 git checkout -b feature/my-feature
 ```
-Then you may run `yarn && yarn start` to start a local server. 
+
+Then you may run `yarn && yarn start` to start a local server.
 
 Once you have made your changes, push your feature branch and make a Pull Request into the dev branch.
```

Comment:
These instructions for making changes are how we do things across all of our repositories, so suggesting we remove it from the readme.